src/portable.rs

```diff
@@ -127,9 +127,7 @@ impl PortableRegistry {
 
             // Make sure any type params are also retained:
             for param in ty.ty.type_params.iter_mut() {
-                let Some(ty) = &param.ty else {
-                    continue
-                };
+                let Some(ty) = &param.ty else { continue };
```

Comment:
hrm, is this the style the rustfmt prefers now? :thinking:

Reply:
Yep, the rustfmt passed on the prior PR and unfortunately after merging it the fmt version changed and now prefers this syntax (from [master/CI job](https://github.com/paritytech/scale-info/actions/runs/5445893612/jobs/9905815377)).